web/src/components/njwds-extended/UnStyledButton.tsx

```diff
@@ -17,10 +17,19 @@ interface Props {
 // eslint-disable-next-line react/display-name
 export const UnStyledButton = forwardRef(
   (props: Props, ref?: React.LegacyRef<HTMLButtonElement>): ReactElement => {
-    const style =
-      props.style === "default"
-        ? "usa-button usa-button--unstyled width-auto font-weight-inherit font-size-inherit"
-        : "usa-button bg-transparent";
+    let style;
+
+    switch (props.style) {
+      case "default":
+        style = "usa-button usa-button--unstyled width-auto font-weight-inherit font-size-inherit";
+        break;
+      case "footerLink":
+        style = "clear-button";
+        break;
+      default:
```

Comment:
[sand] Just noting for posterity here that you will be addressing the "default"/default naming scheme in a future refactor.